src/borg/remote.py

```diff
@@ -236,14 +261,26 @@ def borg_cmd(self, args, testing):
 
     def ssh_cmd(self, location):
         """return a ssh command line that can be prefixed to a borg command line"""
-        args = shlex.split(os.environ.get('BORG_RSH', 'ssh'))
-        if location.port:
-            args += ['-p', str(location.port)]
-        if location.user:
-            args.append('%s@%s' % (location.user, location.host))
+        if sys.platform != 'win32' or shutil.which('ssh') is not None:
+            args = shlex.split(os.environ.get('BORG_RSH', 'ssh'))
```

Comment:
guess the BORG_RSH must be outside the if/elif/else or it will only work if one is not on windows or has ssh installed. but the point of using this env var is if you do NOT just want to use ssh OR if you want ssh, but with different options.